src/WebJobs.Script/Binding/TableBinding.cs

```diff
@@ -184,11 +198,31 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes()
                 }
                 else
                 {
-                    // binding to entire table (query multiple table entities)
-                    RuntimeBindingContext runtimeContext = new RuntimeBindingContext(new TableAttribute(TableName, boundPartitionKey, boundRowKey), additionalAttributes);
+                    // binding to multiple table entities
+                    RuntimeBindingContext runtimeContext = new RuntimeBindingContext(new TableAttribute(TableName), additionalAttributes);
```

Comment:
No bugs. For input bindings, TableAttribute allows binding to either a singleton (specify BOTH PK and RK), or a Queryable. You can't specify just the PK in the attribute and expect that to be automatically applied to the query. If you want to filter, you bind to IQueryable.

In C# that is easy, but in other languages you can't do that.

I'm adding the C# test now, and will verify that things work.